### PR TITLE
New version: Logitech.CameraSettings version 2.12.20

### PR DESCRIPTION
**Pull request by einar-prod**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140210)

### CODE REVIEW

File: manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+PackageIdentifier: Logitech.CameraSettings
+PackageVersion: 2.12.20
+PackageLocale: en-US
+Publisher: Logitech Europe S.A.
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: Logitech Camera Settings
+PackageUrl: https://support.logi.com/hc/en-us/articles/360049055854
```
Comment (stephengillie) [inline manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://support.logi.com/hc/en-us/articles/360056414974-Camera-Settings
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
This URL appears to be paywalled, requring a Logitech account to view. Or maybe that's only when going directly to the URL instead of through a search engine?

**Comment by stephengillie** [comment]
Hi @einar-prod,

> This PR's version number 2.12.20 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 2.12.8.0, which has 4 parameters.

Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)

(Automated response - build 812)

**Comment by einar-prod** [comment]
> Hi @einar-prod,
> 
> > This PR's version number 2.12.20 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 2.12.8.0, which has 4 parameters.
> 
> Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)
> 
> (Automated response - build 812)

Looks like a typo, the correct version number is 2.12.20**.0**

File: manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS0.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Logitech.CameraSettings
+PackageVersion: 2.12.20
```
Comment (stephengillie) [inline manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.installer.yaml:5]:
```suggestion
PackageVersion: 2.12.20.0
```
Please update file name and path to match this change.

File: manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS0.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Logitech.CameraSettings
+PackageVersion: 2.12.20
```
Comment (stephengillie) [inline manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.yaml:5]:
```suggestion
PackageVersion: 2.12.20.0
```
Please update file name and path to match this change.

File: manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS0.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Logitech.CameraSettings
+PackageVersion: 2.12.20
```
Comment (stephengillie) [inline manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.12.20.0
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**